src/test/java/com/semmle/jira/addon/TestCreateIssue.java

```diff
@@ -31,6 +31,8 @@
   CreateValidationResult createValidationResult = mock(CreateValidationResult.class);
   ProcessedConfig config = mock(ProcessedConfig.class);
 
+  private IssueResult issueResult;
```

Comment:
You can initialise it to the mocked value.

Reply:
True, done!